clang/tools/scan-build-py/lib/libscanbuild/arguments.py

```diff
@@ -469,13 +461,15 @@ def parser_build_log(parser):
     parser.add_argument(
         '--parse-build-log',
         metavar='<file>',
-        help="""Specifies the build log file path of project specified by --in-root.""")
+        help="""Specifies the file path of the build log.""")
 
-def parser_in_root(parser):
+def parser_work_directory(parser):
     parser.add_argument(
-        '--in-root',
+        '--work-directory',
         metavar='<path>',
-        help="""Specifies the directory path for the root of the source tree that needs to be migrated.""")
+        default=argparse.SUPPRESS,
+        help="""Specifies the work directory of the source file in the build log that needs to be migrated.
```

Comment:
how about?  (current working-directory is misleading). 
 -S <path-to-source>          = Explicitly specify a source directory.

Reply:
Have synced with Zhiming and Ziran, path-to-source is not accurate, so work-directory is preferred to use.